ignore.js

```diff
@@ -77,21 +77,21 @@ const normalizeOptions = (options = {}) => ({
 	cwd: toPath(options.cwd) || slash(process.cwd()),
 });
 
-export const isGitIgnored = async options => {
+export const isIgnored = async (pattern, options) => {
 	const {cwd} = normalizeOptions(options);
 
-	const paths = await fastGlob('**/.gitignore', {cwd, ...gitignoreGlobOptions});
+	const paths = await fastGlob(pattern, {cwd, ...gitignoreGlobOptions});
 
 	const files = await Promise.all(paths.map(file => getFile(file, cwd)));
 	const ignores = reduceIgnore(files);
 
 	return getIsIgnoredPredicate(ignores, cwd);
 };
 
-export const isGitIgnoredSync = options => {
+export const isIgnoredSync = (pattern, options) => {
 	const {cwd} = normalizeOptions(options);
 
-	const paths = fastGlob.sync('**/.gitignore', {cwd, ...gitignoreGlobOptions});
+	const paths = fastGlob.sync(pattern, {cwd, ...gitignoreGlobOptions});
```

Comment:
Maybe we should add `dot: true` to the option, when pattern is `*ignore`, not sure, need test.

Reply:
> Maybe we should add dot: true to the option, when pattern is `*ignore`

Sorry, I don't understand the suggestion.

> need test.

Added.

Reply:
I mean, if user use `{ignoreFiles: '*ignore'}`, can this glob files starts with `.`? See https://github.com/mrmlnc/fast-glob#dot

Reply:
I personally don't find that necessary, and it makes the configuration a little complicated (what if the user didn't want to use dotfiles?). But I'll leave the decision to you.

Reply:
I'm not a maintainer :)